jobs/MeterInstanceJob.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const _ = require('lodash');
+const logger = require('../common/logger');
+const BaseJob = require('./BaseJob');
+const catalog = require('../common/models/catalog');
+const config = require('../common/config');
+const utils = require('../common/utils');
+const EventLogInterceptor = require('../common/EventLogInterceptor');
+/* jshint ignore:start */
+const maas = require('../data-access-layer/metering');
+/* jshint ignore:end */
+const apiServerClient = require('../data-access-layer/eventmesh').apiServerClient;
+const CONST = require('../common/constants');
+
+class MeterInstanceJob extends BaseJob {
+
+  /* jshint ignore:start */
+  static async run(job, done) {
+    try {
+      logger.info(`-> Starting MeterInstanceJob -  name: ${job.attrs.data[CONST.JOB_NAME_ATTRIB]} - with options: ${JSON.stringify(job.attrs.data)} `);
+      let events = await this.getInstanceEvents()
```

Comment:
Minor comment: Use `const` instead of `let` since the reference is not being changed

Reply:
Done